imports/api/rest/post-process-db-change-message.js

```diff
@@ -16,28 +18,82 @@ export default (req, res) => {
 
   const {
     notification_type: type,
-    invitee_user_id: inviteeId,
     case_title: caseTitle,
     case_id: caseId
   } = message
 
+  var emailAddr, emailContent, assigneeId, assignee
```

Comment:
You should "let" or "const" instead of "var". It still works, but it is inconsistent with the rest of the codebase